lib/index.js

```diff
@@ -46,20 +46,22 @@ module.exports = {
             "ReplyTo": replyTo || defaultReplyTo,
             "TemplateId": templateId,
             "TemplateAlias": templateAlias,
-            "TemplateModel": {
-              ...defaultVariables,
-              ...variables,
-            },
-            ...rest
           };
 
           let method = 'sendEmail';
 
           if (templateId || templateAlias) {
             method = 'sendEmailWithTemplate';
+            msg.TemplateModel = {
+              ...defaultVariables,
+              ...variables,
+            }
           }
 
-          client[method](removeUndefined(msg), function (err) {
+          client[method](removeUndefined({
+            ...msg,
+            ...rest,
```

Comment:
I moved this here, since in theory you could still override the `TemplateModel` in the send options

Reply:
Yeah, I think that makes sense, thanks for the note 🙌🏼